README.md

```diff
@@ -73,6 +75,10 @@ pip install --upgrade git+https://github.com/dhruvdcoder/wandb-allennlp.git@feat
     ```
     export TEST=0
     python slurm_wandb_agent.py <sweep_id> -p baselines -e score-based-learning --num-jobs 5 -f --edit-sbatch --edit-srun
+#SBATCH --exclude=node084,node029,node083,node030,node095
+#SBATCH --exclude=node084,node029,node083,node030,node095
+#SBATCH --exclude=node084,node029,node083,node030,node095
+#SBATCH --exclude=node084,node029,node083,node030,node095
```

Comment:
Should these line(s) be here?

Reply:
I totally forgot to deal with this! (thx for catching this.)